test/automation/src/sql/notebook.ts

```diff
@@ -157,6 +172,17 @@ export class Notebook {
 		await this.code.waitForElement(textSelector, result => result?.textContent === text);
 	}
 
+	async isTextCellInEditMode(): Promise<boolean> {
+		// verify that cell is in editmode
+		const element = await this.code.waitForElement(`.notebook-cell.active ${Notebook.textCellPreviewSelector}`);
+		return element.attributes['contenteditable'] === 'true';
+	}
+
+	async isTextCellActive(): Promise<void> {
```

Comment:
As above: 

1. By convention is functions should always be returning a boolean
2. I would suggest refactoring these so that isTextCellInEditMode calls this function to get the active text cell